Document/0x05d-Testing-Data-Storage.md

```diff
@@ -1016,9 +1016,9 @@ Different checks on the Android device can be implemented by querying different
 Educating users is a crucial part in the usage of mobile Apps. Even though many security controls are already in place, they might be circumvented or misused through the users.
 
 The following list shows potential warnings or advises for a user when opening the App the first time and using it:
-* App shows after starting it the first time a list of data it is storing locally and remotely. This can also be a link to an external ressource as the information might be quite extensive.
+* App shows after starting it is the first time a list of data is storing locally and remotely. This can also be a link to an external ressource as the information might be quite extensive.
```

Comment:
Maybe like this:

App shows a list of data stored locally and remotely. This can also be a link to an external ressource as the information might be quite extensive.

"starting it the first time" is not needed as this is already mentioned in the line before.